dns-zone.tf

```diff
@@ -0,0 +1,12 @@
+data "azurerm_dns_zone" "play" {
+  name                = "us-hpccsystems-dev.azure.lnrsg.io"
+  resource_group_name = "app-dns-prod-eastus2"
+}
+
+resource "azurerm_dns_a_record" "play-record_set" {
+  name = "play-test"
+  zone_name = data.azurerm_dns_zone.play.name
+  resource_group_name = data.azurerm_dns_zone.play.resource_group_name
+  ttl                 = 300
+  records             = ["${data.external.get_eclwatchip.result["ecl_watch_ip"]}"]
```

Comment:
Will have to be retrieved differently once the source of ECL Watch's IP address is changed.

Reply:
> Will have to be retrieved differently once the source of ECL Watch's IP address is changed.
Not sure what you mean by "source of ECL Watch's IP". Currently, I get the ecl watch ip from the the output of "kubectl get services" and expect the IP to be on the line beginning with "eclwatch".